src/Servers/IIS/IIS/test/testassets/InProcessNewShimWebSite/InProcessNewShimWebSite.csproj

```diff
@@ -29,7 +29,7 @@
 
   <ItemGroup>
     <!-- This package is hard-coded to the 2.2.0 package as a part of ensuring ANCM stays forward compatible.  -->
-    <PackageReference Include="Microsoft.AspNetCore.Server.IIS" Version="2.2.0">
+    <PackageReference Include="Microsoft.AspNetCore.Server.IIS" Version="2.2.6">
```

Comment:
This looks like it's changing things to directly conflict w/ the comment. Should we use a completely different version of Server.IIS instead❔

tagging @rafikiassumani-msft as well

Reply:
looks like it was removed before 2.1.0 shipped but was present in 2.2.0 for some reason

Reply:
since 2.2.6 is the newest version that does not put the assembly into our shared framework, should we let this PR use that version❔ the alternative would be to give up on the idea of doing back-compat testing here.

Reply:
Yes, let's drop it. 2.2 is completely out of support and 3.x used a different structure.

Reply:
Took it out. Will follow along enough to react if a test is looking for the old version or some such&hellip;